mission-control/docs/guide/notifications/index.mdx

```diff
@@ -85,6 +85,31 @@ In this example:
 - This expression evaluates to true only when the `type` attribute of the [check](/reference/canary-checker/check) object is equal to `'http'`.
 - So, Mission Control only sends notifications for events that meet this condition, which alerts you when HTTP checks fail.
 
+<details summary="Sophisticated Filters">
+For a more sophisticated filter, you can also use the handy [`matchQuery`](/reference/scripting/cel#matchquery) cel function that supports search query.
+Example:
+
+```yaml title="config.yaml"
```

Comment:
can we reference this ?

Reply:
https://github.com/flanksource/mission-control/pull/1857